ui/lib/core/addon/components/selectable-card-form.hbs

```diff
@@ -0,0 +1,30 @@
+<form class="selectable-card is-rounded no-flex" data-test-selectable-card-form={{@testLabel}}>
+  <div class="is-flex-between is-fullwidth card-details">
+    <h3 class="title is-5">{{@title}}</h3>
+  </div>
+  <div class="has-top-bottom-margin">
+    <p class="sub-text">{{@subText}}</p>
+  </div>
+
+  <div class="is-flex">
+    <SearchSelect
+      class="is-flex-1"
+      @selectLimit="1"
+      @models={{@models}}
+      @backend={{@backend}}
+      @fallbackComponent="input-search"
+      @onChange={{this.handleInput}}
+      @placeholder={{@placeholder}}
```

Comment:
I believe we also want to pass `@disallowNewItems={{true}}` here because I don't think creating a new role is an option here (that form is 🥴 )